src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.QueuesTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,165 @@
+                var queueInvalidName = queueNameGeneration + "$";
+
+                //Invalid QueueName
+                try
+                {
+                    var createinvalidQueueResponse = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueInvalidName,
+                                    new SBQueue() { EnableExpress = true, EnableBatchedOperations = true });
+                }
+                catch (Exception e)
+                {
+                    Assert.Equal("Operation returned an invalid status code 'BadRequest'", e.Message);
+                }
+
+                // Create Queue
+                var queueName = TestUtilities.GenerateName(ServiceBusManagementHelper.QueuesPrefix);
+                var createQueueResponse = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName,
+                new SBQueue() { EnableExpress = true, EnableBatchedOperations = true});
+
+                Assert.NotNull(createQueueResponse);
+                Assert.Equal(createQueueResponse.Name, queueName);
+                Assert.True(createQueueResponse.EnableExpress);
+                
+                // Get the created Queue
+                var getQueueResponse = ServiceBusManagementClient.Queues.Get(resourceGroup, namespaceName, queueName);
+                Assert.NotNull(getQueueResponse);
+                Assert.Equal(EntityStatus.Active, getQueueResponse.Status);
+                Assert.Equal(getQueueResponse.Name, queueName);
+                  
+                // Get all Queues
+                var getQueueListAllResponse = ServiceBusManagementClient.Queues.ListByNamespace(resourceGroup, namespaceName);
+                Assert.NotNull(getQueueListAllResponse);
+                Assert.True(getQueueListAllResponse.Count() >= 1);                
+                Assert.True(getQueueListAllResponse.All(ns => ns.Id.Contains(resourceGroup)));
+
+
+                // Create Queue1
+                var queueName1 = TestUtilities.GenerateName(ServiceBusManagementHelper.QueuesPrefix);
+                var createQueueResponse1 = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName1,
+                new SBQueue() { EnableExpress = true });
+
+
+                // Update Queue. 
+                var updateQueuesParameter = new SBQueue()
+                {
+                    EnableExpress = true,
+                    MaxDeliveryCount = 5,
+                    MaxSizeInMegabytes = 1024,
+                    ForwardTo = queueName1,
+                    ForwardDeadLetteredMessagesTo = queueName1
+                    
+                };
+
+                var updateQueueResponse = ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName, updateQueuesParameter);
+                Assert.NotNull(updateQueueResponse);
```

Comment:
Should verify the properties set during create or update.

Reply:
Done!